http-server/src/main/java/io/airlift/http/server/HttpServerModule.java

```diff
@@ -59,6 +61,21 @@ public class HttpServerModule
 {
     public static final String REALM_NAME = "Airlift";
 
+    private final boolean enableVirtualThreads;
+
+    public HttpServerModule()
+    {
+        this(false);
+    }
+
+    public HttpServerModule(boolean enableVirtualThreads)
```

Comment:
We already have an `HttpServerBinder`, so we should add a method there instead. We could have it bind an annotated boolean value.

Reply:
Makes sense.